### ISSUE TITLE
Fork Linkback Fix

### ISSUE THREAD

**Issue by 0x4007** [devpool-directory/devpool-directory-tasks#45]
> Yes but to clarify this used to work as expected. The problem now is that it seems when the task is assigned and unassigned, it links back even on forks.

So the issue is:
1. New devpool issue is created
2. Fork of https://github.com/ubiquity/devpool-directory creates a new issue with `www` prefix (which is expected) in order not to reference (i.e. spam) the original issue ([example](https://github.com/jordan-ae/devpool-directory/issues/1025))
3. Devpool issue is closed because somebody was assigned
4. Devpool issue is reopened because somebody was unassigned
5. Now the forked issue displays github issue URL without the `www` prefix (while expected behavior to display URL with the `www` prefix) thus spamming original issue ([example](https://github.com/jordan-ae/devpool-directory/issues/867))

---

https://github.com/jordan-ae/devpool-directory/issues/867

When it is reopened, it writes the non `www.` prefixed link to the issue, which "references" the original issue which appears as spam 

<img width="1555" alt="image" src="https://github.com/user-attachments/assets/9534d0a4-365b-4cb9-b335-1db674627140">

![github com_ubiquity_ubiquity-dollar_issues_937](https://github.com/user-attachments/assets/fc1a3abd-c9a7-4588-b966-8e90995875de)

**Comment by rndquu** [comment]
So:
1. https://github.com/ubiquity/devpool-directory should reference github issues without the `www` prefix (which works as expected right now)
2. Forks of https://github.com/ubiquity/devpool-directory should reference github issues with the `www` prefix in order not to spam original issues

Correct?

**Comment by 0x4007** [comment]
Yes but to clarify this used to work as expected. The problem now is that it seems when the task is assigned and unassigned, it links back even on forks.

**Comment by rndquu** [comment]
> Yes but to clarify this used to work as expected. The problem now is that it seems when the task is assigned and unassigned, it links back even on forks.

So the issue is:
1. New devpool issue is created
2. Fork of https://github.com/ubiquity/devpool-directory creates a new issue with `www` prefix (which is expected) in order not to reference (i.e. spam) the original issue ([example](https://github.com/jordan-ae/devpool-directory/issues/1025))
3. Devpool issue is closed because somebody was assigned
4. Devpool issue is reopened because somebody was unassigned
5. Now the forked issue displays github issue URL without the `www` prefix (while expected behavior to display URL with the `www` prefix) thus spamming original issue ([example](https://github.com/jordan-ae/devpool-directory/issues/867))
